app/src/main/res/drawable/continent_oceania.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:viewportWidth="458.9524"
+    android:viewportHeight="454.8092"
+    android:width="458.95242dp"
+    android:height="454.80923dp">

```

Comment:
These should be `64dp` if I remember correctly. Bonus points for using `64px` in inkscape already and centering the drawing in it.